build.gradle

```diff
@@ -19,6 +19,7 @@ buildscript {
         classpath 'com.android.tools.build:gradle:7.0.3'
         classpath "org.jacoco:org.jacoco.core:0.8.7"
         classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.0'
+        classpath 'com.google.gms:google-services:4.3.8'
```

Comment:
@prosper-w3w any reason why you added this dependency?

Reply:
I added the dependency because of firebase test lab, but I can remove it until our app is ready to be deployed on firebase